#!/usr/bin/env python
PACKAGE = "pub_sub_rate"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("loop_rate", int_t, 0, "Publisher loop rate", 10,1,10000)
gen.add("pub_name", str_t, 0, "Publisher Name parameter", "cpp")
gen.add("pub_string", str_t, 0, "String to be Published parameter", "Hello World")

exit(gen.generate(PACKAGE, "pub_sub_rate", "dynamicConfig"))